src/main/java/pl/lukakan/showstracker/cast/person/model/Gender.java

```diff
@@ -0,0 +1,14 @@
+package pl.lukakan.showstracker.cast.person.model;
+
+public enum Gender {
+    MALE("mężczyzna"), FEMALE("kobieta");
+    private final String description;
```

Comment:
description to chyba nienajlepsza nazwa tutaj. Z tego co widzę, to jest nazwa, którą chcesz wyświetlać, a nie opis